lib/moon/components/link.ex

```diff
@@ -34,7 +34,7 @@ defmodule Moon.Components.Link do
         "text-trunks hover:text-piccolo focus:text-bulma active:text-bulma": @optional
       }
       data-phx-link={@phx_link}
-      data-phx-link-state="push"
+      data-phx-link-state={if @replace, do: "replace", else: "push"}
```

Comment:
would  like to see here @phx_link attr (with default: "push") and no @replace attr is required

Reply:
https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#link/1

It either can be replace or push. If it is not replace, it is push.